django/db/__init__.py

```diff
@@ -10,7 +10,7 @@
     'backend', 'connection', 'connections', 'router', 'DatabaseError',
     'IntegrityError', 'InternalError', 'ProgrammingError', 'DataError',
     'NotSupportedError', 'Error', 'InterfaceError', 'OperationalError',
-    'DEFAULT_DB_ALIAS', 'DJANGO_VERSION_PICKLE_KEY'
+    'DEFAULT_DB_ALIAS', 'DJANGO_VERSION_PICKLE_KEY',
```

Comment:
I'd appreciate if changes like this would be left out in the future of feature patches to make them easier to review.